externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala

```diff
@@ -96,7 +96,7 @@ object SparkSQLEngine extends Logging {
     }
 
     val session = SparkSession.builder().config(sparkConf).enableHiveSupport().getOrCreate()
-    session.sql("SHOW DATABASES")
+    kyuubiConf.get(KyuubiConf.ENGINE_INITIALIZE_SQL).split(",").foreach(session.sql)
```

Comment:
may bad, lets use `;`

Reply:
I am not quite sure. Literally, it should support all SQL cases here. But as a matter of fact, we may only excepting some `ADD JAR/FILE` or `CREATE FUNCTION ` here

Reply:
Ok, for these simply use cases, split(";") is sufficient.

And splitSemiColon is a private function and is not a static one, it is difficult to use it gracefully.